compiler/src/iree/compiler/Dialect/Flow/IR/FlowOps.td

```diff
@@ -983,10 +983,19 @@ def FLOW_CallOp : FLOW_Op<"call", [
   let builders = [
     OpBuilder<(ins
       "SymbolRefAttr":$callee,
-      "TypeRange":$resultTypes,
+      "TypeRange":$resultTypes, "ValueRange":$resultDims,
```

Comment:
The `argumentDims` can be inferred/constructed by `buildDynamicDimsForValues`, but I think it's hard for the `resultDims` so I add it to the parameter list. Feel free to comment for any idea here : )